src/tests/ftest/security/cont_create_acl.py

```diff
@@ -0,0 +1,138 @@
+        """
+        acl_args = {"tmp_dir": self.tmp,
+                    "user": self.current_user,
+                    "group": self.current_group,
+                    "permissions": PERMISSIONS}
+
+        ## Getting the default ACL list
+        expected_acl = generate_acl_file("default", acl_args)
+
+        ## 1. Create a pool and obtain its UUID and SVC
+        self.log.info("===> Creating a pool with no ACL file passed")
+        self.pool_uuid, self.pool_svc = self.create_pool_with_dmg()
+
+        ## 2. Create a container with no ACL file passed
+        self.log.info("===> Creating a container with no ACL file passed")
+        self.container_uuid = self.create_container_with_daos(self.pool)
+
+        if not self.container_uuid:
+            self.fail("    An expected container could not be created")
+
+        cont_acl = self.get_container_acl_list(self.pool_uuid,
+                                               self.pool_svc,
+                                               self.container_uuid)
+        if not self.compare_acl_lists(cont_acl, expected_acl):
+            self.fail("    ACL permissions mismatch:\n\t \
+                      Container ACL: {}\n\tExpected ACL: {}".format(
+                          cont_acl, expected_acl))
+        cont_acl = None
+        expected_acl = None
+
+        ## 3. Destroy the container
+        self.log.info("===> Destroying the container")
+        result = self.destroy_containers(self.container)
+        if result:
+            self.fail("    Unable to destroy conatiner '{}'".format(
+                self.container_uuid))
+        else:
+            self.container_uuid = None
+
+        ## Create a valid ACL file
+        self.log.info("===> Generating a valid ACL file")
+        expected_acl = generate_acl_file("valid", acl_args)
+
+        ## 4. Create a container with a valid ACL file passed
+        self.log.info("===> Creating a container with an ACL file passed")
+        self.container_uuid = self.create_container_with_daos(
+            self.pool, "valid")
+
+        if not self.container_uuid:
+            self.fail("    An expected container could not be created")
+
+        cont_acl = self.get_container_acl_list(self.pool_uuid,
+                                               self.pool_svc,
+                                               self.container_uuid,
+                                               True)
+        if not self.compare_acl_lists(cont_acl, expected_acl):
+            self.fail("    ACL permissions mismatch:\n\t \
+                      Container ACL: {}\n\tExpected ACL:  {}".format(
+                          cont_acl, expected_acl))
+        cont_acl = None
+        expected_acl = None
+
+        ## 5. Destroy the container
+        self.log.info("===> Destroying the container")
+        result = self.destroy_containers(self.container)
+        if result:
+            self.fail("    Unable to destroy conatiner '{}'".format(
```

Comment:
(style)  'conatiner' may be misspelled - perhaps 'container'?

Reply:
Done